### ISSUE TITLE
Add 2 new configuration variables to manage oauth providers

### ISSUE THREAD

**Issue by gregmzs** [sameersbn/docker-gitlab#2637]
Shout it be possible to add 2 new configuration variable : 

- SYNC_PROFILE_FROM_PROVIDER (default [])
- SYNC_PROFILE_ATTRIBUTES (default true, but not commented in gitlab.yml anymore)

To manage account existing in Gitlab and a new Azure Authentication.

Thanks ;)

**Comment by flavienbwk** [comment]
Definitely, it looks necessary for [Azure AD v2 configuration](https://serverfault.com/a/943231/450957).
